canonicalwebteam/blog/logic.py

```diff
@@ -97,13 +97,15 @@ def transform_article(
             article["content"]["rendered"]
         )
 
-    if "_start_month" in article:
+    # if _start_month is set here we can assume that day and year
+    # will also have a value, same for the end data
+    if "_start_month" in article and article["_start_month"] != "":
```

Comment:
Can this not just be article.get("start_month")?

Reply:
Thanks! updated it to use `""` as a default value on get, to catch both the None and empty string scenario